cmd/rewards-api/main.go

```diff
@@ -360,7 +361,47 @@ func main() {
 		if err != nil {
 			logger.Fatal().Err(err).Msg("Failed to transfer referral bonuses.")
 		}
+	case "migrate-rewards":
+		var week int
+		if len(os.Args) == 2 {
+			// We have to subtract 1 because we're getting the number of the newly beginning week.
+			week = services.GetWeekNumForCron(time.Now()) - 1
+		} else {
+			var err error
+			week, err = strconv.Atoi(os.Args[2])
+			if err != nil {
+				logger.Fatal().Err(err).Msg("Could not parse week number.")
+			}
+		}
+		logger := logger.With().Int("week", week).Str("sub-command", "migrate-rewards").Logger()
```

Comment:
```suggestion
		logger := logger.With().Int("week", week).Str("subCommand", "migrate-rewards").Logger()
```
I like these to look like "proper JSON".